draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2895,7 +2895,7 @@ https://video.example.com/k/movie1/4/013.ts
 	Example use cases:
 		<list style="symbols">
 		<t>
-		Priority:  uCDN may have multiple trigger commands in “pending” and/or “active” mode. The trigger commands may potentially take a long time to execute, for example due to policy constraints, large number of content objects affected by it or other dCDN business logic.
+		Priority:  uCDN may have multiple trigger commands in “pending” and/or “active” mode. For example, trigger commands with policy constraints, large number of affected content objects affected, or other dCDN business logic may take a long time to execute.
```

Comment:
Two small grammar changes:
large number -> _a_ large number
affected content objects affected -> content objects affected

Reply:
Fixed

Reply:
Resolving; I think you fixed this already.